jbpm-test-util/src/main/java/org/jbpm/test/persistence/scripts/util/TestsUtil.java

```diff
@@ -51,7 +51,7 @@
             final boolean sortByName, final boolean dropFiles) {
         final File folderWithScripts = new File(folderWithDDLs.getPath() + File.separator + databaseType.getScriptsFolderName());
         if (folderWithScripts.exists()) {
-            Predicate<File> filterNew = (name) -> name.getName().toLowerCase().contains("springboot");
+            Predicate<File> filterNew = (name) -> name.getName().toLowerCase().contains("springboot") || name.getName().toLowerCase().contains("bytea");
```

Comment:
Same as above, perhaps better merge with this PR [jbpm#1714](https://github.com/kiegroup/jbpm/pull/1714)

Reply:
done